docs/community/jobcenter.md

```diff
@@ -0,0 +1,43 @@
+---
+title: How to Post A Job
+sidebar_position: 7
+---
+
+# Job Center
+
+The KubeEdge [Job Center](https://kubeedge.io/job-center/) aims to provide a platform for community partners and developers to exchange job opportunities. Partners can post job openings here and provide detailed job information and requirements, while community users can select job positions based on their experience and interests. We sincerely hope that you can find suitable jobs or excellent talents here.
```

Comment:
accessing to https://kubeedge.io/job-center/, image does not show as expected? (off topic from this PR though.)

![image](https://github.com/kubeedge/website/assets/43395114/de8cd6e7-a7e4-45a1-87ae-cb6f9e3f28bb)

Reply:
The image url in demo is wrong, I have fixed it. Check with https://deploy-preview-443--kubeedge.netlify.app/job-center/